src/types.ts

```diff
@@ -70,22 +70,88 @@ export type WorkflowResult = {
   }[];
 };
 
+/**
+ * Internal interface for specifying user utilities for replacement
+ * within a script.  It is built from the different types of user
+ * utilities for a uniform interface.
+ *
+ * @param name - The name of the utility, if it is a function or class
```

Comment:
I believe `@param` only works for functions https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#param-and-returns

You can find some examples at the top of this file for how to document fields on a type/interface 👍

Reply:
Updated